Adjust some headers for glibc 2.12 compatibility.
Patch is a diff between the 1.2.3 release and CVS HEAD.

--- cdrdao-1.2.3/dao/ScsiIf-linux.cc	2009-09-28 05:42:03.000000000 -0430
+++ /home/kkallio/q/src/r/cvs/cdrdao/cdrdao/dao/ScsiIf-linux.cc	2010-08-06 07:50:46.000000000 -0430
@@ -19,6 +19,9 @@
 
 #include <config.h>
 
+#include <sys/types.h>
+#include <sys/stat.h>
+
 #include <stdio.h>
 #include <fcntl.h>
 #include <unistd.h>
